go/worker/common/config/config.go

```diff
@@ -0,0 +1,43 @@
+// Package config implements global configuration options.
+package config
+
+// Config is the common worker configuration structure.
+type Config struct {
+	// Port to use for incoming gRPC client connections.
```

Comment:
Is this still used anywhere? AFAIK everything has migrated to P2P?

Reply:
It is still referenced in `GetNodeAddresses()` in `worker/common/config.go`, but that function does not appear to be used anywhere, so I think it's safe to remove.